cmd/stream-cli/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/GetStream/stream-cli"
+)
+
+func main() {
+	if err := run(); err != nil {
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
+		os.Exit(1)
+	}
+}
+
+func run() error {
+	d, err := os.UserConfigDir()
+	if err != nil {
+		return fmt.Errorf("cannot get user's home directory: %v", err)
+	}
+
+	config, err := cli.NewConfig(d)
```

Comment:
Is this UserConfigDir logic must be in the main file?

Reply:
Yeah for test purposes.

Reply:
ahh I see